include/mbgl/storage/offline.hpp

```diff
@@ -62,10 +63,34 @@ class OfflineGeometryRegionDefinition {
     const float pixelRatio;
 };
 
+/*
+ * An offline region defined by a style URL, list of tiles, and
+ * device pixel ratio.
+ *
+ * Both minZoom and maxZoom must be ≥ 0, and maxZoom must be ≥ minZoom.
+ *
+ * maxZoom may be ∞, in which case for each tile source, the region will include
+ * tiles from minZoom up to the maximum zoom level provided by that source.
+ *
+ * pixelRatio must be ≥ 0 and should typically be 1.0 or 2.0.
+ */
+class OfflineTileListRegionDefinition {
+public:
+  OfflineTileListRegionDefinition(std::string, LatLngBounds, double, double, float, std::vector<CanonicalTileID>);
+  
+  /* Private */
```

Comment:
out of curiosity: what's up with the comment /* Private */ as opposed to actually making the members private (with the private: keyword)? I see the /* Private */ comment is used elsewhere.

Reply:
no idea.